Gems/WhiteBox/Code/Source/EditorWhiteBoxComponent.cpp

```diff
@@ -725,13 +724,13 @@ namespace WhiteBox
         }
 
         // transform ray into local space
-        const AZ::Transform worldFromLocalUniform = AzToolsFramework::TransformUniformScale(m_worldFromLocal);
-        const AZ::Transform localFromWorldUniform = worldFromLocalUniform.GetInverse();
+        const AZ::Transform worldFromLocal = m_worldFromLocal;
+        const AZ::Transform localFromWorld = worldFromLocal.GetInverse();
```

Comment:
[minor] `worldFromLocal` is an unnecessary copy now in this function and it can be removed.

````
const AZ::Transform localFromWorld = m_worldFromLocal.GetInverse();
````